examples/nextjs-spa/tsconfig.json

```diff
@@ -19,6 +20,6 @@
       "@/*": ["./src/*"]
     }
   },
-  "include": ["next-env.d.ts", "**/*.ts", "**/*.tsx"],
-  "exclude": ["node_modules"]
+  "exclude": ["node_modules", "**/*.spec.ts"],
```

Comment:
In cloud, we found it nice to have analytics on the test files, especially for stuff like floating promises, etc. That requires `eslint` to be setup as well, which in turn will require `**/*.spec.ts` to be included AFAIK. Any particular reason it's not right now?

Reply:
Ah not really, was getting errors and though it was due to the spec files being included in the build.